modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,41 @@
+
+* Available: 
+An Operator with the condition type `Available` is functional and available in the cluster. If the status is `False`, at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+An Operator with the condition type `Progressing` is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators do not report the condition type `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the Operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
+
+* Upgradeable:
+An Operator with the condition type `Upgradeable` indicates if the Operator is safe for you to perform a minor or patch update based on the current cluster state. 
++
+The message field contains a human-readable description of what the administrator needs to do for the cluster to successfully update. The CVO allows updates when this condition is `True`, `Unknown` or missing. 
++
+When the `Upgradeable` status is `False`, minor and patch updates are impacted, and the CVO prevents the cluster from performing impacted updates unless forced.
+
+* Failing: 
+The CVO monitors cluster operators and other components. `ClusterStatusFailing` is set on the cluster Version status when a cluster cannot reach the desired state. It is considered more severe than `Degraded`, indicates that the cluster is not healthy, and requires an administrator to intervene.
+
+* Invalid:
+If the the cluster Version has an error that prevents the server from taking action, the condition type of the cluster Version will report as `ClusterVersionInvalid`. The CVO will only reconcile the current state as long as this condition is set.
+
+* RetrievedUpdates:
+The cluster Version condition type `RetrievedUpdates` indicates whether or not available updates have been retrieved from the upstream update server. The condition is `Unknown` before retrieval, `False` if the updates either recently failed or could not be retrieved, or `True` if the `availableUpdates` field is both recent and accurate.
+
+* ReleaseAccepted:
+The cluster Version condition type `DesiredReleaseAccepted` has a `True` status if the requested release payload was successfully loaded without failures during image verification and precondition checking.
+
+* ImplicitlyEnabledCapabilities:
+The cluster Version has the condition type `ImplicitlyEnabledCapabilities` status of `True` if there are enabled capabilities that the user is not currently requesting through `spec.capabilities`. The CVO does not support uninstalling capabilities if any associated resources were previously managed by the CVO.
```

Comment:
Substitute `The CVO does not support uninstalling capabilities` with `The CVO does not support disabling capabilities`